cmd/eib/main.go

```diff
@@ -111,17 +113,17 @@ func validateImageConfigDir(configDir string) error {
 }
 
 func main() {
-	imageConfig, context, err := processArgs()
+	imageConfig, ctx, err := processArgs()
 	if err != nil {
 		zap.L().Fatal("CLI arguments could not be parsed", zap.Error(err))
 	}
 
-	builder := build.New(imageConfig, context)
+	builder := build.New(imageConfig, ctx, combustion.Configure)
```

Comment:
Kept `imageConfig` at least for now even if it's part of the context. We should clear this up in a follow up and once we decide how we'd want the `build` package to look like.